mpqp/core/circuit.py

```diff
@@ -661,18 +749,35 @@ def to_other_language(
         method will be used only for complex objects that are not tractable by
         OpenQASM (like hybrid structures).
 
+        Note:
+            Most providers take noise into account at the job level. A notable
+            exception is Braket, where the noise is contained in the circuit
+            object. For this reason you find the noise included in the Braket
+            circuits.
+
         Args:
             language: Enum representing the target language.
             cirq_proc_id : Identifier of the processor for cirq.
 
         Returns:
             The corresponding circuit in the target language.
 
-        Example:
+        Examples:
             >>> circuit = QCircuit([X(0), CNOT(0, 1)])
             >>> qc = circuit.to_other_language()
             >>> type(qc)
             <class 'qiskit.circuit.quantumcircuit.QuantumCircuit'>
+            >>> circuit2 = QCircuit([H(0), CZ(0,1), Depolarizing(0.6, [0])])
+            >>> braket_circuit = circuit2.to_other_language(Language.BRAKET)
+            >>> print(braket_circuit)  # doctest: +NORMALIZE_WHITESPACE
+            T  : │         0         │         1         │
+                  ┌───┐ ┌───────────┐       ┌───────────┐
+            q0 : ─┤ H ├─┤ DEPO(0.6) ├───●───┤ DEPO(0.6) ├─
+                  └───┘ └───────────┘   │   └───────────┘
+                                      ┌─┴─┐
+            q1 : ─────────────────────┤ Z ├───────────────
```

Comment:
the to_other_language for the noise seem wrong: the noise is applied to all qubits here but it shouldn't be the case

Reply:
Solved